### ISSUE TITLE
Hi @gildas-lormeau, the fix is working for the scenario where I change the folder it is saved to from another folder to the target folder, but there are two situations that seem buggy to me:

### ISSUE THREAD

**Issue by DinoFive** [gildas-lormeau/SingleFile#1178]
Hi @gildas-lormeau, the fix is working for the scenario where I change the folder it is saved to from another folder to the target folder, but there are two situations that seem buggy to me:

1. When I bookmark a page, it starts the save process. If I click _Cancel_ in the bookmark popup the page is still downloaded.
2. When I bookmark a page, it starts the save process. But then I change the _Location_ in the bookmark popup to a different folder. The page is still downloaded even though the bookmark is in the wrong folder. 

These scenarios are less problematic than the one you fixed, as I can manually delete the file myself.

_Originally posted by @willlma in https://github.com/gildas-lormeau/SingleFile/issues/933#issuecomment-1122756432_

**Comment by DinoFive** [comment]
As a solution to what willlma descriped, can SingleFile create a bookmark of the saved webpage to a certain folder automatically when I click the extension Icon to save the webpage?

**Comment by gildas-lormeau** [comment]
@DinoFive Unfortunately, this is not technically possible because SingleFile is not able to know the path of the saved page on the disk.

**Comment by DinoFive** [comment]
> @DinoFive Unfortunately, this is not technically possible because SingleFile is not able to know the path of the saved page on the disk.

@gildas-lormeau SingleFile now can "link the new bookmark to the saved page". How is this possible?

**Comment by gildas-lormeau** [comment]
@DinoFive You're right, I forgot it was possible. You can include a folder name in the `File name > template` option.
